src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-macos`
Can load a native assembly. Possitive test..Can load native dll
```powershell
Exception calling "LoadNative" with "0" argument(s): "Unable to load shared library '/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpXj3DiS.tmp/osx-x64/libSkiaSharp.dylib' or one of its dependencies. In order to help diagnose loading problems, consider setting the DYLD_PRINT_LIBRARIES environment variable: dlopen(/var/folders/24/8k48jl6d249_n_qfxwsl6xvm0000gn/T/tmpXj3DiS.tmp/osx-x64/libSkiaSharp.dylib, 1): image not found"
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 51
51:         [TestNativeClass]::LoadNative() | Should -Not -BeNullOrEmpty

```
Can load a native assembly.Can load native dll
```powershell
Expected an exception, with FullyQualifiedErrorId 'BadImageFormatException' to be thrown, but the FullyQualifiedErrorId was 'DllNotFoundException'. from /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1:115 char:11
    +         { [TestNativeClass2]::LoadNative() } | Should -Throw -ErrorId ?
    +           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
at <ScriptBlock>, /Users/runner/runners/2.160.1/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 115
115:         { [TestNativeClass2]::LoadNative() } | Should -Throw -ErrorId "BadImageFormatException"
```